src/repository/noteRelations.repository.ts

```diff
@@ -47,4 +47,22 @@ export default class NoteRelationsRepository {
   public async getParentNoteIdByNoteId(noteId: NoteInternalId): Promise<number | null> {
     return await this.storage.getParentNoteIdByNoteId(noteId);
   }
+
+  /**
+   * Delete all note ralations contains noteId
+   *
+   * @param noteId - id of the current note
+   */
+  public async deleteNoteRelationsByNoteId(noteId: NoteInternalId): Promise<boolean> {
+    return await this.storage.deleteNoteRelationsByNoteId(noteId);
```

Comment:
maybe return deleted note? then we could chenk exactly which noteRelation has been deleted

Reply:
The `destroy` function returns either the number of deleted items - I can't return an object that doesn't exist
![image](https://github.com/codex-team/notes.api/assets/94054053/4ec5c25b-da24-4de3-b006-f766ef36c5f2)